packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -446,61 +491,17 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 	});
 
 	describe("Deleted data stores in summary", () => {
-		/**
-		 * Validates that the given data store state is correct in the summary.
-		 * e.g. if expectDelete is true::
-		 * - It should be deleted from the data store summary tree.
-		 * - It should not be present in the GC state in GC summary tree.
-		 * - It should be present in the deleted nodes in GC summary tree.
-		 *
-		 * And the opposite results if false.
-		 */
-		function validateDataStoreStateInSummary(
-			summaryTree: ISummaryTree,
-			dataStoreNodePath: string,
-			expectDelete: boolean = true,
-		) {
-			const shouldShouldNot = expectDelete ? "should" : "should not";
-
-			// Check if the data store is deleted from the data store summary tree or not.
-			const deletedDataStoreId = dataStoreNodePath.split("/")[1];
-			const channelsTree = (summaryTree.tree[channelsTreeName] as ISummaryTree).tree;
-			assert.notEqual(
-				Object.keys(channelsTree).includes(deletedDataStoreId),
-				expectDelete,
-				`Data store ${deletedDataStoreId} ${shouldShouldNot} have been deleted from the summary`,
-			);
-
-			// Validate that the GC state does not contain an entry for the deleted data store.
-			const gcState = getGCStateFromSummary(summaryTree);
-			assert(gcState !== undefined, "GC tree is not available in the summary");
-			assert.notEqual(
-				Object.keys(gcState.gcNodes).includes(dataStoreNodePath),
-				expectDelete,
-				`Data store ${dataStoreNodePath} ${shouldShouldNot} have been removed from GC state`,
-			);
-
-			// Validate that the deleted nodes in the GC data has the deleted data store.
-			const deletedNodesState = getGCDeletedStateFromSummary(summaryTree);
-			assert.equal(
-				deletedNodesState?.includes(dataStoreNodePath) ?? false,
-				expectDelete,
-				`Data store ${dataStoreNodePath} ${shouldShouldNot} be in deleted nodes`,
-			);
-		}
-
 		it("updates deleted data store state in the summary", async () => {
 			const { unreferencedId, summarizer } =
-				await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
+				await summarizationWithUnreferencedDataStoreAfterTime();
 			const sweepReadyDataStoreNodePath = `/${unreferencedId}`;
-			await sendOpToUpdateSummaryTimestampToNow(summarizer);
 
 			// Summarize. In this summary, the gc op will be sent with the deleted data store id. The data store
 			// will be removed in the subsequent summary.
-			await summarizeNow(summarizer);
+			await summarize(summarizer);
```

Comment:
Maybe rename to `syncAndSummarize` or something?  In the past I've gotten tripped up on the lack of `ensureSynchronized` before when reading tests in this file I think

Reply:
Renamed to `ensureSynchronizedAndSummarize`